Signal/src/ViewControllers/ConversationView/Cells/OWSMessageBubbleView.m

```diff
@@ -404,9 +404,9 @@ - (void)configureViews
         [self.viewConstraints addObjectsFromArray:@[
             [bodyTextView autoPinLeadingToSuperviewMarginWithInset:textInsets.leading],
             [bodyTextView autoPinTrailingToSuperviewMarginWithInset:textInsets.trailing],
+            [bodyTextView autoSetDimension:ALDimensionWidth toSize:bodyTextContentSize.width],
+            [bodyTextView autoSetDimension:ALDimensionHeight toSize:bodyTextContentSize.height],
         ]];
-        [self.viewConstraints
-            addObject:[bodyTextView autoSetDimension:ALDimensionHeight toSize:bodyTextContentSize.height]];
```

Comment:
This probably isn't how we'll want to do things in the end (to support messages with more than one of (body text, media, quoted reply, etc.) we'll probably rework these constraints a bit.  But this will fix some obvious breakage until I get there.